### PR TITLE
bip8: tiny wording fixups

### PR DESCRIPTION
**Pull request by darosior**
RE-reading through the BIP i found some sentences to make it seem like LOT was always `true` but it was made optional.

### CODE REVIEW

File: bip-0008.mediawiki [hunk 1]
```diff
@@ -16,13 +16,13 @@
 
 This document specifies an alternative to [[bip-0009.mediawiki|BIP9]] that corrects for a number of perceived mistakes.
 Block heights are used for start and timeout rather than POSIX timestamps.
-It additionally introduces an additional activation parameter to guarantee activation of backward-compatible changes (further called "soft forks").
+It additionally introduces an optional activation parameter to guarantee activation of backward-compatible changes (further called "soft forks").
```
Comment (michaelfolkson) [inline bip-0008.mediawiki:19]: The parameter isn't optional. What you set the parameter to is optional. I would rather:

"It additionally introduces an ~additional~ activation parameter that **can** guarantee activation of backward-compatible changes"

**Review by michaelfolkson — COMMENTED** [review]
I do think the wording could be sharpened but I think the first suggested change isn't optimal either. Happy with the second suggested change.

File: bip-0008.mediawiki (hunk 1 above)
Comment (darosior) [reply bip-0008.mediawiki]: Thanks, added!

**Comment by michaelfolkson** [comment]
ACK

**Pull request merged**